spec/rubocop/cop/style/mixin_usage_spec.rb

```diff
@@ -58,6 +68,28 @@ class C
       RUBY
     end
 
+    it 'registers an offense when using `include` in method definition ' \
```

Comment:
does *not* register :-)

Reply:
Oops! I fixed it 💦

Reply:
There was a slight misunderstanding. This was good with `registers`, the other was not good. https://github.com/bbatsov/rubocop/pull/5272#discussion_r157417483

Thank you for pointing out.